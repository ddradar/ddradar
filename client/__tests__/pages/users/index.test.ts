import { createLocalVue, mount } from '@vue/test-utils'
import Buefy from 'buefy'

import UserListPage from '~/pages/users/index.vue'

const localVue = createLocalVue()
localVue.use(Buefy)

describe('pages/users/index.vue', () => {
  test('renders correctly', () => {
    const wrapper = mount(UserListPage, { localVue })
    expect(wrapper).toMatchSnapshot()
  })
})
