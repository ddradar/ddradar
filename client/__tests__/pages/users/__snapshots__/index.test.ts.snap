// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/users/index.vue renders correctly 1`] = `
<section class="section">
  <section>
    <div class="field is-grouped-multiline is-grouped">
      <!---->
      <div class="field"><label class="label">所属地域</label>
        <div class="control"><span class="select"><select><!----> <option value="0">
            未指定
          </option><option value="1">
            北海道
          </option><option value="2">
            青森県
          </option><option value="3">
            岩手県
          </option><option value="4">
            宮城県
          </option><option value="5">
            秋田県
          </option><option value="6">
            山形県
          </option><option value="7">
            福島県
          </option><option value="8">
            茨城県
          </option><option value="9">
            栃木県
          </option><option value="10">
            群馬県
          </option><option value="11">
            埼玉県
          </option><option value="12">
            千葉県
          </option><option value="13">
            東京都
          </option><option value="14">
            神奈川県
          </option><option value="15">
            新潟県
          </option><option value="16">
            富山県
          </option><option value="17">
            石川県
          </option><option value="18">
            福井県
          </option><option value="19">
            山梨県
          </option><option value="20">
            長野県
          </option><option value="21">
            岐阜県
          </option><option value="22">
            静岡県
          </option><option value="23">
            愛知県
          </option><option value="24">
            三重県
          </option><option value="25">
            滋賀県
          </option><option value="26">
            京都府
          </option><option value="27">
            大阪府
          </option><option value="28">
            兵庫県
          </option><option value="29">
            奈良県
          </option><option value="30">
            和歌山県
          </option><option value="31">
            鳥取県
          </option><option value="32">
            島根県
          </option><option value="33">
            岡山県
          </option><option value="34">
            広島県
          </option><option value="35">
            山口県
          </option><option value="36">
            徳島県
          </option><option value="37">
            香川県
          </option><option value="38">
            愛媛県
          </option><option value="39">
            高知県
          </option><option value="40">
            福岡県
          </option><option value="41">
            佐賀県
          </option><option value="42">
            長崎県
          </option><option value="43">
            熊本県
          </option><option value="44">
            大分県
          </option><option value="45">
            宮崎県
          </option><option value="46">
            鹿児島県
          </option><option value="47">
            沖縄県
          </option><option value="48">
            香港
          </option><option value="49">
            韓国
          </option><option value="50">
            台湾
          </option><option value="51">
            アメリカ
          </option><option value="52">
            ヨーロッパ
          </option><option value="53">
            海外
          </option><option value="106">
            カナダ
          </option><option value="107">
            シンガポール
          </option><option value="108">
            タイ
          </option><option value="109">
            オーストラリア
          </option><option value="110">
            ニュージーランド
          </option><option value="111">
            イギリス
          </option><option value="112">
            イタリア
          </option><option value="113">
            スペイン
          </option><option value="114">
            ドイツ
          </option><option value="115">
            フランス
          </option><option value="116">
            ポルトガル
          </option><option value="117">
            インドネシア
          </option><option value="118">
            フィリピン
          </option></select></span>
          <!---->
        </div>
        <!---->
      </div>
      <div class="field"><label class="label">登録名(部分一致)</label>
        <div class="control is-clearfix"><input type="text" autocomplete="on" class="input">
          <!---->
          <!---->
          <!---->
        </div>
        <!---->
      </div>
      <div class="field"><label class="label">DDR CODE</label>
        <div class="control is-clearfix"><input type="text" autocomplete="on" maxlength="8" placeholder="10000000" minlength="8" pattern="^\\d{8}$" class="input">
          <!---->
          <!----> <small class="help counter is-invisible">
            0 / 8
          </small></div>
        <!---->
      </div>
      <!---->
    </div>
    <div class="field">
      <!----> <button type="button" class="button is-success">
        <!----> <span>
        検索
      </span>
        <!----></button>
      <!---->
    </div>
  </section>
  <section>
    <div class="b-table">
      <!---->
      <!---->
      <div class="table-wrapper">
        <table class="table is-striped">
          <!---->
          <tbody>
            <tr class="is-empty">
              <td colspan="0">
                <section class="section">
                  <div class="content has-text-grey has-text-centered">
                    <p>Nothing here.</p>
                  </div>
                </section>
              </td>
            </tr>
          </tbody>
          <!---->
        </table>
      </div>
      <div class="level">
        <div class="level-left"></div>
        <div class="level-right">
          <div class="level-item">
            <nav class="pagination"><a role="button" href="#" disabled="disabled" class="pagination-link pagination-previous"><span class="icon" aria-hidden="true"><i class="mdi mdi-chevron-left mdi-24px"></i></span></a> <a role="button" href="#" disabled="disabled" class="pagination-link pagination-next"><span class="icon" aria-hidden="true"><i class="mdi mdi-chevron-right mdi-24px"></i></span></a>
              <ul class="pagination-list">
                <!---->
                <!---->
                <!---->
                <!---->
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </div>
  </section>
</section>
`;
